File: VL09/VL09.py
def sovlegiaithua(end_index: int):
    if end_index < 0:
        return None
    tich = 1
    for i in range(1, end_index + 1):
        tich *= i
    return tich


x, n = input().split()
x = float(x)
n = int(n)

S = 0.0
for i in range(1, n + 1):
    S += (x ** i) / sovlegiaithua(i)

print(f"{S:.2f}")

File: VL09/VL09.cpp
#include<bits/stdc++.h>
using namespace std;

long long sovlegiaithua(int end_index) {
    if (end_index < 0) return -1;
    long long tich = 1;
    for (int i = 1; i <= end_index; i++) {
        tich *= i;
    }
    return tich;
}

int main() {
    string x_str;
    int n;
    cin >> x_str >> n;
    double x = stod(x_str);
    double S = 0.0;
    for (int i = 1; i <= n; i++) {
        double power = 1.0;
        for (int j = 0; j < i; j++) {
            power *= x;
        }
        S += power / sovlegiaithua(i);
    }
    cout << fixed << setprecision(2) << S << endl;
    return 0;
}
